containers/base/Dockerfile.in

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:jessie
+FROM _base_image_
```

Comment:
Instead of a template, let's make this a local Docker tag (e.g. `datalab-base-image`), and then whatever the base is will be based on that tag rather than having to munge template files.